talk/owt/sdk/ic/selfiesegmentationmodel.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OWT_IC_SELFIESEGMENTATIONMODEL_H_
+#define OWT_IC_SELFIESEGMENTATIONMODEL_H_
+
+#include <string>
+#include <vector>
+
+#include "inference_engine.hpp"
+#include "opencv2/core/mat.hpp"
+
+namespace owt {
+namespace ic {
+
+class SelfieSegmentationModel {
+ public:
+  explicit SelfieSegmentationModel(InferenceEngine::Core& core);
+
+  bool LoadModel(const std::string& xmlPath, const std::string& device = "CPU");
+  bool IsLoaded() const;
+
+  cv::Mat predict(const cv::Mat& frame);
+  void predictAsync(const cv::Mat& frame);
+  cv::Mat waitForFinished();
```

Comment:
what's the point of public this?

Reply:
Sync/Async prediction are options provided by openvino. I just pass on these to provides potential caller with free usage.

Reply:
I don't agree. If you want to provide async API,  your API should include a callback param instead of asking developer to call another API just to wait for the result.

Reply:
The `waitForFinished` function behave like `await Promise`, I believe this is another async call design except for callbacks. Anyway, we may remove the async api since the postprocessors use a sequential processing.